.github/review/values.yml

```diff
@@ -113,3 +113,5 @@ config:
     scope: openid
   sentry:
     dsn: https://3de59e3a93034a348089131aa565bdf4@sentry.data.amsterdam.nl/27
+  frontPageAlert:
+      text: Text of alert
```

Comment:
I wasn't really sure what this file does (didn't give any hits on google) but all the other keys in the app.amsterdam.json were there as well. I figured I should add this one too.

Reply:
Can't see my comment anywhere so if this is double, my apologies.

I don't really know what this file does (the internet doesn't return any useful hints on search terms github yaml review or related search terms) but I figured, since all the other configs of app.amsterdam.json are there, I should add frontPageAlert too.

Reply:
Better to find out what you are doing instead of just doing things you're not sure about. You never know what you change. If you don't know, you can always aks :)
I think this is the config file that will be used when review environments are spinned up. So you can remove it here.

Reply:
Will do. I've removed it.